src/proxy.conf

```diff
@@ -12,8 +12,11 @@
   RewriteCond %{HTTP:Upgrade} !=websocket
   RewriteRule /EC2_INSTANCE_ID/(.*)     http://localhost:8787/$1 [P,L]
 
-  <Location /EC2_INSTANCE_ID/ >
+  <LocationMatch /EC2_INSTANCE_ID/ >
```

Comment:
new config and original (commit de5c45ef42cfde83ae7bfdd99b651ce34e064300) configs both seem to work.  decided to revert back to original because it's been used  and tested more than new config.